pkg/ddl/backfilling_test.go

```diff
@@ -362,53 +362,6 @@ func assertDistSQLCtxEqual(t *testing.T, expected *distsqlctx.DistSQLContext, ac
 	require.Equal(t, errctx.NewContextWithLevels(expected.ErrCtx.LevelMap(), expected.WarnHandler), actual.ErrCtx)
 }
 
-// TestReorgExprContext is used in refactor stage to make sure the newDefaultReorgDistSQLCtx() is
-// compatible with newMockReorgSessCtx(nil).GetDistSQLCtx() to make it safe to replace `mock.Context` usage.
-// After refactor, the TestReorgExprContext can be removed.
```

Comment:
It's removed because we are "After refactor", and the `newMockReorgSessCtx(nil).GetDistSQLCtx()` is actually different from the `newDefaultReorgDistSQLCtx`, because we add a `RuntimeStatsColl` for the `newDefaultReorgDistSQLCtx`